APIs/openEO/Python_Client/Python.qmd

```diff
@@ -0,0 +1,346 @@
+
+
+### Processes
+
+Processes in openEO are operations that can be applied on (EO) data
+(e.g. calculate the mean of an array, or mask out observations outside a given polygon).
+The output of one process can be used as the input of another process,
+and by doing so, multiple processes can be connected that way in a larger "process graph":
+a new (user-defined) processes that implements a certain algorithm.
+
+::: {.callout-note}
+
+Check [the openEO glossary](../Glossary.qmd)
+for more details on pre-defined, user-defined processes and process graphs.
+
+:::
+
+For other graphical overviews of the openEO processes, there is an [online documentation](https://docs.openeo.cloud/processes/){target="_blank"} for general process descriptions and the [openEO Hub](https://hub.openeo.org/) for back-end specific process descriptions. 
+
+## Authentication 
+
+In the code snippets above we did not need to log in
+since we just queried publicly available back-end information.
+However, to run non-trivial processing queries one has to authenticate
+so that permissions, resource usage, etc. can be managed properly.
+
+
+A detailed description of why and how to use the authentication methods is on the [official documentation](https://open-eo.github.io/openeo-python-client/auth.html#authentication-and-account-management){target="_blank"}.
+
+
+To authenticate your account on the backend of the Copernicus Data Space Ecosystem, it is necessary for you to complete the registration process. Once registered, you will be authenticated for executing processes.
+
+The following code snippet shows how to log in via OIDC authentication:
+
+```python
+print("Authenticate with OIDC authentication")
+connection.authenticate_oidc()
+```
+The method is used to authenticate yourself on the back-end authentication system. 
+Calling this method with either will print a URL to visit, that opens your system web browser where you can use your Copernicus Data Space Ecosystem credentials, or when there are refresh tokens, these will be used directly.
+After that the website will give you the instructions to go back to the python client, where your connection has logged your account in. 
+This means that every call that comes after that via the `connection` variable is executed by your user account.
+
+
+## Working with Datacube
+
+Now that we know how to discover the capabilities of the back-end and how to authenticate, let's do some real work and process some EO data in a batch job. We'll build the desired algorithm by working on so-called "Datacubes", which is the central concept in openEO to represent EO data.
+
+
+### Creating a Datacube
+
+The first step is loading the desired slice of a data collection
+with [`Connection.load_collection`](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.load_collection){target="_blank"}:
+
+```python
+datacube = connection.load_collection(
+        "SENTINEL1_GRD",
```

Comment:
I'm not sure if this will work. Sentinel-1 grd is special, because you always need to convert to backscatter.
I would use Sentinel-2 L2A as an introductory example.

Reply:
Updated example with available EVI calculation example